nio/schemas.py

```diff
@@ -1925,3 +1925,65 @@ class Schemas:
             "state_key",
         ],
     }
+
+    space_hierarchy = {
```

Comment:
I'll check for them later, but I'm wondering if this particular schema could safely refer to other schemas already defined in the file (and whether that would impact readability)